### DISCUSSION TITLE
Problem with domain name

### DISCUSSION THREAD

**Discussion by DorianLaguna** [domcloud/domcloud#60] [Q&A]
When I create a new domain i set the name. But when server is created the domain change:
![image](https://github.com/domcloud/domcloud/assets/123023143/8751d2eb-81ae-4783-ba9a-f5d19a3e5f45)
![image](https://github.com/domcloud/domcloud/assets/123023143/3f06b0a9-8127-42fc-9638-c4ad8c4fcdd2)

**Comment by willnode** [comment]
This is expected, please take a look at the link below. It's a recent change.

https://github.com/domcloud/domcloud/issues/58#issuecomment-1745947814

I just recently made some changes in the UI to avoid this confusion.

To quote[ the updated docs](https://domcloud.co/docs/intro/getting-started#right-side):

> Using a free domain has its own limitations: The name can't be customized and cannot have custom records. This is intended to prevent malicious use of our platform.
